testConnectivity.py

```diff
@@ -1196,10 +1227,10 @@ def testConnectivity(args):
             exit(os.EX_USAGE)
 
     # Enforce a minimum number of phase scans
-    if args.nPhaseScans < 50:
-        printYellow("You've requested the number of phase scans to be {0} which is less than 50.\nThis is probably not reliable, reseting to 50".format(args.nPhaseScans))
-        args.nPhaseScans = 50
-        pass
+    #if args.nPhaseScans < 50:
+    #    printYellow("You've requested the number of phase scans to be {0} which is less than 50.\nThis is probably not reliable, reseting to 50".format(args.nPhaseScans))
+    #    args.nPhaseScans = 50
+    #    pass
```

Comment:
if, as in @bdorney's comment above, we're removing commented lines, here's a candidate